modules/ossm-extensions-migrating-to-wasmplugin.adoc

```diff
@@ -0,0 +1,30 @@
+////
+This module included in the following assemblies:
+*service_mesh_/v2x/ossm-extensions.adoc
+////
+:_content-type: PROCEDURE
+[id="ossm-extensions-migrating-to-wasmplugin_{context}"]
+= Migrating to `WasmPlugin` resources
+
+To upgrade your WebAssembly extensions from the `ServiceMeshExtension` API to the `WasmPlugin` API, you rename your plugin file.
+
+.Prerequisites
+
+* `ServiceMeshControlPlane` upgraded to version 2.2 or later.
+
+[CAUTION]
+====
+Because both plugins will be invoked for every request, you might want to remove your existing `ServiceMeshExtension` resource before creating the new `WasmPlugin` resource. You might get undesired results having two plugins active at the same time.
+====
+
+.Procedure
+
+. Update your container image. If the plugin is already in `/plugin.wasm` inside the container, skip to the next step.  If not:
+
+.. Ensure the plugin file is named `plugin.wasm`. Extension files must be named `plugin.wasm`.
```

Comment:
Global comment: Consider adding "file" after specifying the file name. 
Here's the recommendation in the IBM Style Guide: 
_"Files and directories": Generally, do not use a file name, file name extension, file type, or directory name as a noun. Instead, use the file name, file name extension, file type, or directory name as an adjective. (See page 207)_